### ISSUE TITLE
Brainstorming

### ISSUE THREAD

**Issue by ijlyttle** [uncoast-unconf/uu-2019-day-zero#1]
What should we do?

**Comment by sctyner** [comment]
My thoughts: 

- How to write an R package (many people probably won't have done this before)
- How to collaborate with Github  (same as above) 
- How to tidy data 
- Intro to tidyverse packages & principles

**Comment by ijlyttle** [comment]
I just got finished reading Sam's [excellent blog post](https://sctyner.github.io/rstudio-conf.html), and I was thinking about point 5: "R users don’t have to become R developers."

My preconceived notion of the unconf was that we would be working on *package* projects, and that day-zero could be made to support that.

One way we could support Sam's point is to have an additional area of emphasis, say, on workflows. If we did, could this suggest two tracks on day-zero, perhaps a morning where we work on common topics like git, pull-requests, etc. - then an afternoon that could be a mini-Tidyverse Developers Day for those interested in following the package track, and something more workflow related for that track (if it were a workflow track).

I have to confess that my brain is wired to think in terms of packages. That does not diminish my enthusiasm to support other tracks, just my ability to come up with good ideas for it.

Bottom line, could we - for example - provide two tracks in the afternoon?
